instrumentation/opentelemetry-instrumentation-mysqlclient/tests/test_mysqlclient_integration.py

```diff
@@ -96,6 +97,262 @@ def test_instrument_connection(self, mock_connect):
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
 
+    @mock.patch("opentelemetry.instrumentation.dbapi.instrument_connection")
+    @mock.patch("MySQLdb.connect")
+    # pylint: disable=unused-argument
+    def test_instrument_connection_enable_commenter_dbapi_kwargs(
+        self,
+        mock_connect,
+        mock_instrument_connection,
+    ):
+        cnx = MySQLdb.connect(database="test")
+        cnx = MySQLClientInstrumentor().instrument_connection(
+            cnx,
+            enable_commenter=True,
+            commenter_options={"foo": True},
+        )
+        cursor = cnx.cursor()
+        cursor.execute("Select 1;")
+        kwargs = mock_instrument_connection.call_args[1]
+        self.assertEqual(kwargs["enable_commenter"], True)
+        self.assertEqual(kwargs["commenter_options"], {"foo": True})
+
+    def test_instrument_connection_with_dbapi_sqlcomment_enabled(self):
+        mock_connect_module = mock.MagicMock(
+            __name__="MySQLdb",
+            threadsafety="123",
+            apilevel="123",
+            paramstyle="test",
+        )
+        mock_connect_module._mysql.get_client_info.return_value = "foobaz"
+        mock_cursor = mock_connect_module.connect().cursor()
+        mock_cursor.execute = mock.MagicMock()
```

Comment:
I don't think this line is needed (and in the other tests)

Reply:
Removed in [740df4b](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2941/commits/740df4b61458a01544c886a7054b219d574bc374)